locodedb/record.go

```diff
@@ -0,0 +1,50 @@
+package locodedb
+
+import (
+	"fmt"
+)
+
+// Key represents the key in NeoFS location database.
```

Comment:
Just location DB, not necessarily NeoFS.

Reply:
I deleted NeoFS nearly everywhere, except README,  should I rewrite README now or late?